aria-practices.html

```diff
@@ -712,31 +712,30 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
     <section class="widget" id="combobox">
       <h3>Combo Box</h3>
```

Comment:
The spelling of combo box (2 words) vs combobox (1 word) is inconsistent. I prefer combobox.

Reply:
Changed in https://github.com/w3c/aria-practices/pull/1255/commits/f82e215c70ec084dde67eb3c60eef4e34bd09415